app/crud/metric.py

```diff
@@ -17,7 +17,10 @@ def get_all_by_ids(self, metric_ids: set):
         metrics = self.db_session.query(Metric).filter(
             Metric.id.in_(metric_ids)).all()
         return metrics
-
+    
+    def get_metric_by_id(self, metric_id: int):
```

Comment:
this is useless, there's a function called 'get' already exists in the CRUDMetric due to inheritance.